RELEASE/scripts/sl_ascend/sl_combat.ash

```diff
@@ -3129,18 +3127,6 @@ string sl_edCombatHandler(int round, string opp, string text)
 		return "skill " + $skill[Fist Of The Mummy];
 	}
 
-
-#	if(!contains_text(combatState, "love stinkbug") && sl_have_skill($skill[Summon Love Stinkbug]) && (mcd <= 50))
-#	{
-#		set_property("sl_combatHandler", combatState + "(love stinkbug1)");
-#		return "skill " + $skill[Summon Love Stinkbug];
-#	}
-#	if(!contains_text(combatState, "love stinkbug") && get_property("lovebugsUnlocked").to_boolean() && (mcd <= 50))
-#	{
-#		set_property("sl_combatHandler", combatState + "(love stinkbug2)");
-#		return "skill " + $skill[Summon Love Stinkbug];
-#	}
-
```

Comment:
I'd err towards leaving this commented-out over deleting it. It looks like passable stinkbug logic, even if stinkbug has rotated.

Reply:
I'm also unclear on why the MP restore logic cared about whether you have stinkbug, since I think all the stinkbug skills are free. Weird.

Reply:
That would probably be because you won't need as much mp restore stuff if you have lovebugs, because the damage + hp heal bug is replaced with a damage + mp heal bug in Ed.

Reply:
Not a fan of keeping commented out code in source control. It kind of defeats the purpose of source control. I can see why Cheesecookie might have had it around when he was the only maintainer but it adds unnecessary noise to the codebase if there are multiple people working on it.